packages/blast-tests/e2e-tests/tests/node-start-status.test.sh

```diff
@@ -29,12 +33,28 @@ if [[ $exit_status == 1 ]]; then
         sleep $timer
     done;
 fi
-cd template
+cd $init_folder
+
 if [[ ! `blast node status` =~ 'online' ]]; then
     echo -e $FAILED
     exit_status=1
 else
     echo -e $PASSED
 fi
 
+echo -n 'blast node status -n [network]...'
+# Setup config to allow local node connection (and therefore successful run) only for passed --network parameter
+# invalidate [localNetwork] and [defaultNetwork] values and add valid local network to [networks]
+sed -i '' $'s|localNetwork: \'http://localhost:26657\'|localNetwork: \'https://sentry1.gcp-uscentral1.cudos.org:26657\'|' blast.config.js
+sed -i '' $'s|defaultNetwork: \'\'|defaultNetwork: \'https://sentry1.gcp-uscentral1.cudos.org:26657\'|' blast.config.js
+sed -i '' $'s|networks: {|networks: {\tlocalhost_test: \'http://localhost:26657\',|' blast.config.js
+
+if [[ ! `blast node status -n localhost_test` =~ 'online' ]]; then
```

Comment:
what if the testnet is down?

Reply:
usually the tests are not responsible for the status of public networks or other external dependencies, they shouldn't even make real calls to the testnet

Reply:
the idea is to change the default networks to something that will fail the test so that I can test that the passed parameter is taken into consideration when the test pass.
It is not very clear what I am doing at first glance so I will replace the testnet with something more obvious for test failure.